portal-api/src/v1/gef/models/facilities.js

```diff
@@ -210,6 +212,14 @@ class Facility {
           throw new InvalidParameterError('isUsingFacilityEndDate', req.isUsingFacilityEndDate);
         }
         this.isUsingFacilityEndDate = req.isUsingFacilityEndDate;
+        if (req.facilityEndDate != null) {
```

Comment:
Might want more validation here (throw error if invalid pair of values - isUsignFacilityEndDate = true with bankReviewDate